src/configuration/configuration-base.js

```diff
@@ -128,6 +128,10 @@ export default class Configuration {
         return this._parseConfigurationFileContent(configurationFileContent);
     }
 
+    async _isConfigurationFileExists () {
+        return await Configuration._isConfigurationFileExists(this.filePath);
```

Comment:
Remove `static` keyword for  `Configuration._isConfigurationFileExists`  method.

Reply:
of course